test

```diff
@@ -33,9 +33,13 @@ while [ $# -gt 0 ]; do
             shift 1
             ;;
         "-netgo")
-            TAGS="-tags netgo"
+            TAGS="netgo"
             shift 1
             ;;
+        "-tags")
+            TAGS="$2"
```

Comment:
Yes; essentially `-netgo` is a special case of `-tags`

The point of `"${TAGS[@]}"` is to preserve quoting, since tags is a space-separated list (thanks, Go).
Maybe there is a neater way to achieve that.